torch/_dynamo/output_graph.py

```diff
@@ -477,6 +477,9 @@ def __init__(
         )
 
         self.guard_on_key_order: set[str] = set()
+        self.saved_tensors_hooks_subgraph_names: Optional[list[str]] = (
```

Comment:
Maybe worth a comment explaining what this is in a bit more detail.

In particular, these are the ambient, currently-global saved_tensor_hooks stashed in autograd that are set for the entire duration of the compiled region. This is an invariant today because we graph break on the `saved_tensor_hook` context manager inside a compiled region

Reply:
Yes, will add the comment.
Just did not fully understand.
What did you mean by "because we graph break on the saved_tensor_hook context manager inside a compiled region"?